### ISSUE TITLE
Add new Fields

### ISSUE THREAD

**Issue by cpommier** [plantbreeding/API#486]
New Study fields
```
study.species: [
      "http://aims.fao.org/aos/agrovoc/c_8504"
    ]
In addition to commonCropnames. It brings controled vocabularies plus multispeces managment
```

```
Add list of available experimental factors to the Study:
"experimentFactors": [
      {
        "factor": "fertilizer",
        "modality": "low fertilizer"
      }, {
        "factor": "fertilizer",
        "modality": "High input"
      }
    ]
```

```
Add URI to  experimental factors :
"experimentFactors": [
      {
        "factor": "fertilizer",
         "factorPUI": "test-factor:dia2020-2-nv.irrigation",
        "modality": "low fertilizer",
        "modalityPUI": "test-factor:dia2020-2-nv.lf"
      }, {
        "factor": "fertilizer",
         "factorPUI": "test-factor:dia2020-2-nv.irrigation",
        "modality": "High input"
        "modalityPUI": "test-factor:dia2020-2-nv.irrigation.ww"
      }
    ]
```

```
Projects mapped to brapi.program ? TBC? Make this explicit
    "projects": [
      {
        "uri": "test-isa-prj:eppn2020",
        "name": "EPPN2020",
        "rdf_type": "vocabulary:Project",
        "rdf_type_name": "project"
      }
    ]
```

```
At the study level
One on Phenoarch (HTP greenhouse), another at diaphen (htp field), and a final one at Mons (Classical Pheno)
Semanticaly close to Location, but nonetheless sligthly diferent ?
    "organisations": [
      {
        "uri": "test-isa-infra:alsia",
        "name": "ALSIA",
        "rdf_type": "vocabulary:NationalInfrastructure",
        "rdf_type_name": "national infrastructure"
      }
    ]
```

```
In ObservationVariables:
"trait.attributePUI":"PO:00012345",
"trait.entityPUI":"PATO:00098765"
```

```
In ObservationVVariables:
"observationVariableDescription":"lorem Ipsum"
```

```
https://app.swaggerhub.com/apis/PlantBreedingAPI/BrAPI-Germplasm/2.0#/Germplasm/get_germplasm
same as pedigree
seedSource | stringexample: A0000001/A0000002

```

```
Create separated fileds in addition to subtaxa ?
"subsp." (for subspecies); "convar." (for convariety); "var." (for variety); "f." (for form); "Group" (for "cultivar group")
subspecies, cultivar, variety, subvariety, convariety, group, subgroup, hybrid, line, form, subform 
```

**Comment by cpommier** [comment]
@cpommier : Identify project fields that would be needed in the program object.

**Comment by cpommier** [comment]
Done, see #488

**Comment by marieALaporte** [comment]
```
In ObservationVariables:
"trait.attributePUI":"PO:00012345",
"trait.entityPUI":"PATO:00098765"
```

I do not think that these fields should be linked to an ontology ID simply because not all the traits can be easily split into the Entity/attribute pattern. Often, you end up needing more than one ID to describe each field. 
The Entity - Attribute fields were added to the CO TD in order to facilitate the definition of a trait. It worked in practice when someone is not sure if 2 similar traits are in fact one trait, or should be kept separated. But these fields for more complicated traits (e.g. leaf carbon content, seed dry weight, disease x susceptibility, ...) are not recorded in a consistent way depending on the ontology, which is a slightly different issue, but related in a way.

**Comment by cpommier** [comment]
Hi @marieALaporte , the entity attribute are not always present/desirable, I totally agree. And they can be very messy. But they are increasingly used for defining traits and are therefore needed as optional fields.

**Comment by BrapiCoordinatorSelby** [comment]
I have divide this issue up into its parts so they can be handled separately. #488, #541, and #510 will be fixed as part of the V2.1 release. I feel the others need more thought and discussion before they are ready to be added to the standard.
